Drawsana Demo/ViewController.swift

```diff
@@ -199,6 +199,22 @@ class ViewController: UIViewController {
     drawingView.userSettings.fontName = "Marker Felt"
     applyUndoViewState()
   }
+  
+  override func viewDidLayoutSubviews() {
```

Comment:
This will add a new shape every time the view lays out its subviews, so it will potentially add infinite shapes.

https://developer.apple.com/documentation/uikit/uiviewcontroller/1621398-viewdidlayoutsubviews